server.js

```diff
@@ -1,10 +1,34 @@
 const express = require('express'); // remember to install your npm packages
+const cors = require('cors');
+const helmet = require('helmet');
+const mongoose = require('mongoose');
+const Budget = require('./Budget/budget.js');
```

Comment:
We don't need to import our Mongoose schemas here. `Budget`, `Category` and `Expense` are all unused variables.